src/service/tokens.py

```diff
@@ -0,0 +1,23 @@
+"""
+Functions for creating and hashing tokens.
+"""
+
+import hashlib
+import secrets
+
+
+def create_token(prefix=None):
+    """
+    Create a URL safe token using the default python bit size for secrets.
+
+    prefix - an optional prefix to apply to the token string.
+    """
+    prefix = prefix if prefix else ""
+    return "coll-selection-" + secrets.token_urlsafe()  # 256 bits by default
```

Comment:
So this is a bug. That being said, with the md5ification of the selection IDs I'm working on right now, this file gets entirely deleted, so it's moot